submissions/AbrahamJudah/HTML&CSS/styles.css

```diff
@@ -0,0 +1,68 @@
+body {
```

Comment:
This file's name is similar to the other stylesheet. Please rename the file after the page it styles